frontend/src/concepts/pipelines/content/configurePipelinesServer/PipelineCachingSection.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+  FormGroup,
+  Checkbox,
+  Alert,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import FormSection from '#~/components/pf-overrides/FormSection';
+
+type PipelineCachingSectionProps = {
+  enableCaching: boolean;
+  setEnableCaching: (value: boolean) => void;
+  variant?: 'form' | 'description';
+};
+
+export const PipelineCachingSection = ({
+  enableCaching,
+  setEnableCaching,
+  variant = 'form',
+}: PipelineCachingSectionProps): React.JSX.Element => {
+  const alertRef = React.useRef<HTMLDivElement>(null);
+
+  const [userClickedCheckbox, setUserClickedCheckbox] = React.useState(false);
+
+  React.useEffect(() => {
+    // When caching is disabled (alert appears), scroll to it;
+    // but only if it is unchecked because the user just unchecked it
+    // (don't auto-scroll in the manage (edit) modal)
+    if (userClickedCheckbox && !enableCaching && alertRef.current) {
+      alertRef.current.scrollIntoView({
+        behavior: 'smooth',
+        block: 'nearest',
+      });
+    }
+  }, [enableCaching, userClickedCheckbox]);
+
+  const checkboxElement = (
+    <Checkbox
+      id="pipeline-enable-caching-checkbox"
+      data-testid="pipeline-cache-enabling"
+      name="enable-caching-checkbox"
+      label="Allow caching to be configured per pipeline and task"
+      isChecked={enableCaching}
+      onChange={() => {
+        setEnableCaching(!enableCaching);
+        setUserClickedCheckbox(true);
+      }}
+      body="When enabled, pipelines and tasks can configure caching. By default, caching is on unless changed. This setting can only be changed later in DSPA."
+    />
+  );
```

Comment:
please match the same ux that was done in the section above for the body
<img width="1129" height="383" alt="Screenshot 2025-08-04 at 2 19 27 PM" src="https://github.com/user-attachments/assets/bbf7cd98-8be7-4152-b3c5-456234ce46cd" />

Reply:
done:
<img width="797" height="273" alt="Screenshot 2025-08-04 at 4 11 18 PM" src="https://github.com/user-attachments/assets/f2fcbc3d-14c2-4dfe-8b95-66bb4d35ba26" />